app/scripts/util/utils.js

```diff
@@ -33,4 +35,25 @@ function isValidMSISDN(inputtxt){
       return false;
     }
   }
+}
+
+
+
+function beautifyIndent(type, content){
+  
+  if ( type.indexOf('text/xml') >= 0 || type.indexOf('application/xml') >= 0 ){
```

Comment:
Could you please add the following as well?
`application/soap+xml`
`application/xop+xml`

Although maybe we should start thinking about making it more generic, maybe with a regular expression check for application/stuff+xml, but it's not too critical for now I think.